docs/src/simulation_tips.md

```diff
@@ -260,22 +259,22 @@ without that step we get an error:
 
 ```julia
 julia> model.velocities.u.data
-3×3×3 OffsetArray(::CUDA.CuArray{Float64,3}, 0:2, 0:2, 0:2) with eltype Float64 with indices 0:2×0:2×0:2:
+3×3×3 OffsetArray(::CuArray{Float64, 3, CUDA.Mem.DeviceBuffer}, 0:2, 0:2, 0:2) with eltype Float64 with indices 0:2×0:2×0:2:
 [:, :, 0] =
-Error showing value of type OffsetArrays.OffsetArray{Float64,3,CUDA.CuArray{Float64,3}}:
-ERROR: scalar getindex is disallowed
+Error showing value of type OffsetArrays.OffsetArray{Float64, 3, CuArray{Float64, 3, CUDA.Mem.DeviceBuffer}}:
+ERROR: Scalar indexing is disallowed.
 ```
 
 Here `CUDA.jl` throws an error because scalar `getindex` is not `allowed`. Another way around 
 this limitation is to allow scalar operations on `CuArray`s. We can temporarily
-do that with the `CUDA.@allowscalar` macro or by calling `CUDA.allowscalar(true)`.
+do that with the `CUDA.@allowscalar` macro or by calling `CUDA.allowscalar() do .. end`.
 
 
 ```julia
-julia> using CUDA; CUDA.allowscalar(true)
+julia> using CUDA
 
-julia> model.velocities.u.data
-3×3×3 OffsetArray(::CuArray{Float64,3}, 0:2, 0:2, 0:2) with eltype Float64 with indices 0:2×0:2×0:2:
+julia> CUDA.@allowscalar model.velocities.u.data
+3×3×3 OffsetArray(::CuArray{Float64, 3, CUDA.Mem.DeviceBuffer}, 0:2, 0:2, 0:2) with eltype Float64 with indices 0:2×0:2×0:2:
```

Comment:
@tomchor I tried this and it was **not** working on my HPC with Julia v1.7.

Does this works for you on Julia v1.6? Do we need to update anything here?

Reply:
It definitely worked for me when I wrote this with 1.6. I'll test that next time I get access to a GPU (should be this upcoming week), but feel free to replace it for something that works on 1.7 for you!